Magento2/Tests/Legacy/CopyrightUnitTest.php

```diff
@@ -0,0 +1,33 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+namespace Magento2\Tests\Legacy;
+
+use PHP_CodeSniffer\Tests\Standards\AbstractSniffUnitTest;
+
+class CopyrightUnitTest extends AbstractSniffUnitTest
+{
+    /**
+     * @inheritdoc
+     */
+    public function getErrorList(): array
+    {
+        return [];
+    }
+
+    /**
+     * @inheritdoc
+     */
+    public function getWarningList($testFile = ''): array
+    {
+        if ($testFile === 'CopyrightUnitTest.4.inc') {
+            return [];
+        }
+        
+        return [
+            null => 1,
```

Comment:
just want to confirm why... are you using `null` to indicate that the whole file is incorrect?

Reply:
When I did, it I was thinking in the case you don't have the comment at all, but when you have it and it is not correct may be is better to add the line. I will do the changes.